ui/app/src/Silences/Types.elm

```diff
@@ -37,6 +44,12 @@ type alias Silence =
     , updatedAt : Time
     , matchers : List Matcher
     , silencedAlerts : ApiData (List Alert)
+    , status : SilenceStatus
+    }
+
+
+type alias SilenceStatus =
```

Comment:
Since this is already in `module Silences.Types`, I would prefer it just be `Status` to avoid some stutter.